src/main/resources/init_db.sql

```diff
@@ -1,20 +1,59 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
+CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8 DEFAULT ENCRYPTION='N';
+USE
+    `taxi`;
 
 DROP TABLE IF EXISTS `manufacturers`;
```

Comment:
this wont work if there is any data in ur tables as far as u will have rows referencing on ones from this table
same belov